stackslib/src/chainstate/burn/operations/mod.rs

```diff
@@ -39,9 +39,9 @@ use crate::chainstate::burn::{ConsensusHash, Opcodes};
 use crate::chainstate::stacks::address::PoxAddress;
 use crate::util_lib::db::{DBConn, DBTx, Error as db_error};
 
+/// This module contains all burn-chain operations
```

Comment:
I _think_ this needs to be an self rustdoc (`//!`), because otherwise this rustdoc is getting associated with `delegate_stx`, but admittedly, I've never really gotten those to work.

Reply:
I'll try `//!`